src/vue-apollo.js

```diff
@@ -25,6 +29,37 @@ const authLink = setContext((_, { headers }) => {
   };
 });
 
+const errorLink = onError(({ graphQLErrors, networkError }) => {
+
+  //if we're in production, just show generic message without details
+  if (process.env.NODE_ENV === 'production') {
+    Vue.notify({
+          group: "error",
+          title: i18n.t('oops'),
+          text: i18n.t('msgSomethingWentWrong'),
+          type: "error"
+        });
+    return;
+  }
```

Comment:
The API should return an error code so we could show a translated and detailed message.